File: jokercsi/Programmers/H-Index.py
def solution(citations):
    answer = 0
    
    n = len(citations)
    
    for i in range(1, n+1):
        count = 0
        for j in citations:
            if j >= i:
                count += 1
        if i >= count:
            answer = count
            break
    
    return answer

File: Sjaize/Sorting/H-index.java
// https://school.programmers.co.kr/learn/courses/30/lessons/42747
import java.util.Arrays;

class Solution {
    public int solution(int[] citations) {
        Arrays.sort(citations);
        
        for (int h = citations.length; h >= 1; h--) {
            int index = citations.length - h;
            if (citations[index] >= h) {
                return h;
            }
        }
        return 0;
    }
}
